temporalio/worker/workflow_sandbox/_runner.py

```diff
@@ -52,9 +53,12 @@
 )
 
 
+@dataclass
 class SandboxedWorkflowRunner(WorkflowRunner):
     """Runner for workflows in a sandbox."""
 
+    restrictions: SandboxRestrictions = SandboxRestrictions.default
+
     def __init__(
```

Comment:
I think you can remove this `__init__` and make `runner_class` and `worker_level_failure_exception_types` as properties, with the latter as a `field(default_factory=list, init=False)`. We'll need to keep the docs for each property.